bindata/network/ovn-kubernetes/microshift/role.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: openshift-ovn-kubernetes-node
+  namespace: openshift-ovn-kubernetes
+rules:
+- apiGroups: [""]
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups: [certificates.k8s.io]
+  resources: ['certificatesigningrequests']
+  verbs:
+    - create
+    - get
+    - delete
+    - update
+    - list
```

Comment:
Why do we need this?

Reply:
maybe it shall be moved to cluster role per https://github.com/openshift/cluster-network-operator/pull/1463 ?

Reply:
done